wsidicom/instance.py

```diff
@@ -57,20 +58,39 @@ class Requirement(IntEnum):
 @dataclass
 class WsiAttribute:
     requirement: Requirement
+    image_types: Tuple[str, ...] = ()
     default: Any = None
 
-    def get_default(self) -> Any:
-        if self.requirement == Requirement.ALWAYS:
-            raise ValueError('Attribute is set as always required')
+    def get_default(self, image_type: str) -> Any:
+        if (
+            self.requirement == Requirement.ALWAYS
+            and image_type in self.image_types
+        ):
+            raise ValueError(
```

Comment:
Would it be worth adding a custom error type for the strict checking, so that a user of the library (e.g. WP2) can parse a file in strict mode, and catch the custom error type and classify it for the end user?

`WsiDicomStrictRequirementError`?

Reply:
Yes. Also added error for WsiDicomRequirementError